sample/config

```diff
@@ -2,6 +2,8 @@
 # Space-separated list of source directories to backup.
 source_directories: /home /etc
 
+# one_file_system: 1
```

Comment:
This example might be clearer if the value was `true`. See: https://docs.python.org/2/library/configparser.html#ConfigParser.RawConfigParser.getboolean